core/bindings.rs

```diff
@@ -531,7 +531,7 @@ fn ref_op<'s>(
   let mut state = state_rc.borrow_mut();
 
   let promise_id = match v8::Local::<v8::Integer>::try_from(args.get(0))
-    .map(|l| l.value() as PromiseId)
+    .map(|l| l.uint32_value(scope).unwrap_or_default() as PromiseId)
```

Comment:
This is default to 0 if no arg is passed, we should probably throw an error explicitly instead of defaulting to id 0

Reply:
With the move to `#[op(v8)]` non-number arguments will throw, but negative numbers might be truncated to 0. That's probably not worth fixing though.